modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/StarRatingView.kt

```diff
@@ -26,12 +26,14 @@ import au.com.shiftyjelly.pocketcasts.compose.theme
 import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.PodcastRatingsViewModel
 import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.PodcastRatingsViewModel.RatingState
 import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.PodcastRatingsViewModel.Star
+import au.com.shiftyjelly.pocketcasts.ui.helper.FragmentHostListener
 import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
 import au.com.shiftyjelly.pocketcasts.utils.extensions.abbreviated
 import java.util.UUID
 
 @Composable
 fun StarRatingView(
+    fragmentHostListener: FragmentHostListener,
```

Comment:
Passing a `FragmentHostListener` into a composable feels a little weird to me, but since this composable should always open the same thing anytime the it is clicked, passing in something like `onClick: () -> Unit` didn't feel right since we would always be passing in the same on click behavior.